packages/db/src/index.ts

```diff
@@ -194,7 +194,7 @@
  * <summary>Example usage</summary>
  *
  * ```typescript
- * import type { Provider } from "web3/providers";
+ * import type { Web3BaseProvider as LegacyProvider } from "web3-common";
```

Comment:
I think this should be:

` * import type { Web3BaseProvider } from "web3-types";`

or if it needs to be `LegacyProvider` then:

` * import type { LegacyRequestProvider } from "web3-types";` or `LegacySendProvider`/`LegacySendAsyncProvider`